docs/user-guide/templates.md

```diff
@@ -360,7 +381,78 @@ jobs:
 ```
 It becomes unclear whether the user was trying to override `preinstall` or wrap `install`.
 
-### Writing a screwdriver.yaml for your template repo
+#### Using a template
```

Comment:
there is an existing title with same name, so table of contents is linking there instead of here. Better just call this `Using Template composition`